src/main/java/lottoauto/domain/lotto/Lottos.java

```diff
@@ -0,0 +1,58 @@
+package lottoauto.domain.lotto;
+
+import lottoauto.domain.WinnerBoard;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
+
+public class Lottos {
+
+    private final List<Lotto> lottoList;
+    public static final int EACH_LOTTO_PRICE = 1000;
+
+    public Lottos(List<Lotto> lottoList) {
+        this.lottoList = lottoList;
+    }
+
+    public WinnerBoard checkWinnerLotto(String winNumbersString) {
```

Comment:
당첨 번호를 메서드 내부에서 Lotto 타입으로 생성하고 있는데요
인자의 타입을 Lotto로 변경하면 어떨까요?
```suggestion
    public WinnerBoard checkWinnerLotto(Lotto winNumbers) {
```

Reply:
피드백 감사합니다. 개인적으로 질문이 있습니다.

매개변수를 Lotto로 변경하게 되면 LottoAuto 클래스에서 inputView를 통해 받은 사용자 입력을 Lotto 클래스로 한번 변환해야 하는 과정이 있는데 (현재는 checkWinnerLotto 내부에서 진행하고 있는 부분입니다.)

Lotto 클래스로 변환하는 클래스를 추가로 만드는게 좋을까요?? 아니면 inputView에서 조작을 하는게 좋을까요?? 혹은 LottoAuto 클래스에서 변환해서 넘기는게 좋을까요??

저는 view에서는 다른 로직 없이 입력받은 값을 그대로 전달하는게 좋다고 생각해서 그대로 넘긴 후에 checkWinnerLotto 내부에서 Lotto 클래스로 만들어줬는데 리뷰어님은 어느 부분에 입력받은 문자를 Lotto로 변환하는 작업을 진행하도록 수정하실지 궁금합니다!!

감사합니다.

Reply:
특정 방식이 좋다는 정답은 없는 것 같아요 😃 
저라면 컨트롤러 역할을 하는 LottoAuto 에서 변환할 것 같은데요
컨트롤러가 직접 변환을 할지 변환하는 클래스(DTO)를 활용할지는 옵션인 것 같아요

Reply:
피드백 감사합니다!!
클래스를 활용할지 직접 변환할지 좀더 고민해보고 다음 미션에 같이 적용하겠습니다!!